client/a8c-for-agencies/sections/sites/sites-dashboard/style.scss

```diff
@@ -394,6 +393,15 @@
 					}
 				}
 
+				// Ideally instead of reusing the site name full field
+				// We should be setting a media field and a primary field.
+				.dataviews-view-list__media-wrapper {
```

Comment:
I'll have to check more, but hiding fields with css doesn't seem like something good.

Reply:
Yes, but that's how it was done before. What we need to do here is instead build a "media field" and a "primary field" but that's not how it was done and I don't want to do that in this PR.